### PR TITLE
Implement support for test environment

### PR DESCRIPTION
**Pull request by cdiddy77**
There needs to be a way for test artists to interact with the test environment, for (wait for it) testing purposes. This will allow them (us) to test mint test NFTs that can be bought and sold on test blockchains.

Test and prod environments use different auth providers, so the user would need to re-login or all their server calls will fail.

### CODE REVIEW

**Comment by cdiddy77** [comment]
My change eff-d up all the formatting. How do I make so the formatting follows correct style? (@chornesays )

File: Editor/DevPanel.cs [hunk 1]
```diff
@@ -230,13 +280,15 @@ public class DevPanel : EditorWindow {
 
         private Dictionary<string, Bundle> _bundles = new Dictionary<string, Bundle>();
         private EventSourceReader _evt;
-        private async void GetFiltersOnQueue() {
+        private async void GetFiltersOnQueue()
+        {
             _bundles = new Dictionary<string, Bundle>();
             string getUrlQueue = $"https://filta-machina.firebaseio.com/bundle_queue.json?orderBy=\"artistId\"&equalTo=\"{loginData.localId}\"&print=pretty";
```
Comment (gazuntype) [inline Editor/DevPanel.cs:286]: would need to change this to test version of firebase too right?

File: Editor/DevPanel.cs [hunk 2]
```diff
@@ -245,50 +297,67 @@ public class DevPanel : EditorWindow {
             ListenToQueue();
         }
 
-        private void ListenToQueue() {
+        private void ListenToQueue()
+        {
             _evt = new EventSourceReader(new Uri($"https://filta-machina.firebaseio.com/bundle_queue.json?orderBy=\"artistId\"&equalTo=\"{loginData.localId}\"")).Start();
```
Comment (gazuntype) [inline Editor/DevPanel.cs:302]: would also need to change this? we can just take out the base url and store it in a variable like earlier done

**Review by gazuntype — COMMENTED** [review]
some url change required

**Comment by chornesays** [comment]
> My change eff-d up all the formatting. How do I make so the formatting follows correct style? (@chornesays )

DMd the omnisharp.json file that has formatting

would love to code review - having trouble with all the formatting changes

File: Editor/DevPanel.cs (hunk 1 above)
Comment (cdiddy77) [reply Editor/DevPanel.cs]: Fixed

**Review by chornesays — APPROVED** [review]
Yep. I like this. I'll probably change some copy when I drop in for some other stuff but I'm just happy this actually works.

One thing that is super misleading -- prod users wouldnt be able to log into test and would probably be hella confused. We need to keep all users in sync from prod-->test

**Comment by cdiddy77** [comment]
> Yep. I like this. I'll probably change some copy when I drop in for some other stuff but I'm just happy this actually works.
> 
> One thing that is super misleading -- prod users wouldnt be able to log into test and would probably be hella confused. We need to keep all users in sync from prod-->test

Yeah, so I feel like if you have a checkbox for "(advanced) use local firebase host" you are already confusing users. 

FWIW I have the ability to [copy all the user accounts en masse, or just a select few](https://github.com/getfilta/nplus1/blob/main/tools/packages/firebase-util/src/importUsers.ts) and also I copied all of the users over when I did the initial population.

**Comment by chornesays** [comment]
> Yeah, so I feel like if you have a checkbox for "(advanced) use local firebase host"
I'm already planning on removing that next time I hop in the code _because_ its confusing to users

That said - I wonder if we run your script as a cron at a regular cadence.

**Pull request merged**